tests/phpt/out_of_memory.phpt

```diff
@@ -20,23 +21,18 @@ while (!file_exists($vendor . '/vendor')) {
 require $vendor . '/vendor/autoload.php';
 
 init([
-    'dsn' => 'http://public:secret@local.host/1',
-    'send_attempts' => 1,
+    'before_send' => function (Event $event): ?Event {
+        Assert::assertTrue($event->getLevel()->isEqualTo(Severity::fatal()));
+        $error = $event->getExceptions()[0];
```

Comment:
Please add an assert to check that the array is not empty. Also, I would move all the remaining asserts below this line